### PR TITLE
Feature/67 extending pod overview

### PR DESCRIPTION
**Pull request by rati3l**

### CODE REVIEW

File: _ui/src/clients/response_types/autogen.tsx [hunk 1]
```diff
@@ -3,6 +3,18 @@ export type ActiveCronjobInfo = {
     name: string,
     namespace: string,
 }
+export type ByConfigMapName = ConfigMap[]
```
Comment (perber) [inline _ui/src/clients/response_types/autogen.tsx:6]: Please exclude this ByThings in the hack script. 
You could do it with the param -x or just find a way to remove everything starting with By?

File: _ui/src/config/index.tsx [hunk 2]
```diff
@@ -1 +1 @@
-export const KDD_BASE_URL="http://localhost:3333"
\ No newline at end of file
+export const KDD_BASE_URL="http://nzxt:3333"
```
Comment (perber) [inline _ui/src/config/index.tsx:1]: should we excluded the config and only add an example for now? 
like config/index.example.tsx

**Review by perber — CHANGES_REQUESTED** [review]

File: _ui/src/config/index.tsx (hunk 2 above)
Comment (rati3l) [reply _ui/src/config/index.tsx:1]: either that or i'll open some ports for you on my router... I've forgot to change that back :D

**Review by perber — APPROVED** [review]
lgtm

**Pull request merged**